site/guide/working-with-model-inventory.qmd

```diff
@@ -0,0 +1,51 @@
+---
+title: "Working with the model inventory"
+date: last-modified
+listing:
+  - id: model-inventory-listing
+    type: grid
+    max-description-length: 250
+    sort: false
+    fields: [title, description]
+    contents:
+      - register-models-in-model-inventory.qmd
+      - edit-model-inventory-fields.qmd
+      
+---
+The model inventory holds all models. Within the model inventory, register new models and edit fields. Organize the inventory by searching, filtering and sorting models. Configure the inventory by managing column layout and choosing card or table view.
```

Comment:
This needs to be reworked to be less than 240 characters... because it's going to show up truncated on the card under the Guides page! (By the way, make sure to adjust the Guides page to the updated page navigation, i.e. hide the "Register" and "Edit" as they should be wrapped into the "Working with the model inventory" landing page)